cpp/test/test_postprocessor.cc

```diff
@@ -56,3 +60,76 @@ BOOST_AUTO_TEST_CASE(test_compute_energy)
     BOOST_REQUIRE(std::abs(error[i]) < tolerance);
   }
 }
+
+// Check that the postprocessor output the quantities that were asked
+BOOST_AUTO_TEST_CASE(test_supercapacitor_inspector)
+{
+  // Build an energy storage device
+  boost::property_tree::ptree ptree;
+  boost::property_tree::info_parser::read_info("super_capacitor.info", ptree);
+  ptree.put("debug.material_ids", true);
+  ptree.put("debug.material_properties",
+            "solid_electrical_conductivity,liquid_electrical_conductivity");
+  ptree.put("debug.solution_fields",
+            "solid_potential,liquid_potential,overpotential");
+  ptree.put("debug.solution_fluxes",
+            "solid_current_density,liquid_current_density");
+  boost::mpi::communicator world;
+  std::shared_ptr<cap::EnergyStorageDevice> device =
+      cap::EnergyStorageDevice::build(ptree, world);
+
+  // Execute one time step
+  double const time_step = 10.;
+  double const imposed_current = 1e-2;
+  device->evolve_one_time_step_constant_current(time_step, imposed_current);
+
+  // Output the vtu file
+  cap::SuperCapacitorInspector<2> supercap_inspector;
+  supercap_inspector.inspect(device.get());
+
+  // Check for that the fields are present in the output file
+  unsigned int fields_found = 0;
+  std::vector<std::string> fields;
+  fields.push_back("material_id");
+  fields.push_back("liquid_electrical_conductivity");
+  fields.push_back("solid_electrical_conductivity");
+  fields.push_back("solid_potential");
+  fields.push_back("liquid_potential");
+  fields.push_back("overpotential");
+  fields.push_back("solid_current_density_0");
+  fields.push_back("solid_current_density_1");
+  fields.push_back("liquid_current_density_0");
+  fields.push_back("liquid_current_density_1");
```

Comment:
I don't want to try every possible combinations because I can also output every single material property...